### PR TITLE
Update windows-10-1903-removed-features.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3777
Added note.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Minor copy editing is needed for this PR. Thanks.

File: windows/deployment/planning/windows-10-1903-removed-features.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ The following features and functionalities are removed from the installed produc
...
 | Desktop messaging app doesn't offer messages sync |                                                                                                                          The messaging app on Desktop has a sync feature that can be used to sync SMS text messages received from Windows Mobile and keep a copy of them on the Desktop. The sync feature has been removed from all devices. Due to this change, you will only be able to access messages from the device that received the message.                                                                                                                          |
 
+> [!NOTE]
+> Android devices with the 'Your Phone' app, can sync text messages over mobile data and MMS.
```
Comment (JohanFreelancer9) [inline windows/deployment/planning/windows-10-1903-removed-features.md]:
```suggestion
> Android devices with the **Your Phone** app installed can sync text messages over mobile data and MMS.
```

**Comment by e0i** [comment]
@Reeced40 Please follow-up with copy editing suggestions. Thanks.

**Comment by e0i** [comment]
@greg-lindsay The PR that addresses issue #3777 that has passed copy review is available for review. Thanks.

**Comment by greg-lindsay** [comment]
This note has no bearing on the topic. The topic applies to Windows 10, not Android.

**Pull request closed without merging**